geps/gep-2648/index.md

```diff
@@ -0,0 +1,410 @@
+  upstream Condition type
+- MUST use the `targetRef` struct to specify their target resource.
+- MUST follow the naming requirements (MUST be named to clearly indicate that the
+kind is a Policy, and SHOULD use the `Policy` suffix at the end of the Kind and
+`policies` at the end of the Resource names).
+- MAY target a subsection of a resource using the `sectionName` field of the
+  `targetRef` struct. If it does, it MUST abide by the rules in this document,
+  and MUST clearly indicate the objects that `sectionName` may be used on in its
+  documentation.
+
+## Direct Policy Attachment
+
+A Direct Policy Attachment is tightly bound to one instance of a particular
+Kind within a single namespace (or to an instance of a single Kind at cluster scope),
+and only modifies the behavior of the object that matches its binding.
+
+As an example, the BackendTLSPolicy is specified in GEP-1897, BackendTLSPolicy -
+Explicit Backend TLS Connection Configuration. This Policy attaches to the
+Service object and tells Gateway API implementations what TLS settings should be
+used to connect to that Service when it is used as a backend by a Route.
+
+See GEP-1897 for all the details of this Policy object.
+
+## Direct Policy design guidelines
+
+With this example in mind, here are some guidelines for when to consider
+using Direct Policy Attachment:
+
+* The number or scope of objects is exactly _one_ object.
+* The modifications to be made to the objects don’t have any transitive information -
+  that is, the modifications MUST only affect the single object that the targeted
+  metaresource is bound to, and MUST NOT have ramifications that flow beyond that
+  object.
+* In terms of status, it SHOULD be reasonably easy for a user to understand that
+  everything is working - basically, as long as the targeted object exists, and
+  the modifications are valid, the metaresource is valid, and this should be
+  straightforward to communicate in one or two Conditions. The `status` stanza
+  in BackendTLSPolicy is an example of one of the recommended ways to achieve this.
+* Direct Policy Attachment SHOULD only be used to target objects in the same
+  namespace as the Policy object. Allowing cross-namespace references brings in
+  significant security concerns, and/or difficulties about merging cross-namespace
+  policy objects. Notably, Mesh use cases may need to do something like this for
+  consumer policies, but in general, Policy objects that modify the behavior of
+  things outside their own namespace should be avoided unless it uses a handshake
+  of some sort, where the things outside the namespace can opt–out of the behavior.
+  (Notably, this is the design that we used for ReferenceGrant).
+
+## Apply Policies to Sections of a Resource
+
+The `sectionName` field of `targetRef` can be used to target a specific section of other resources:
+
+* Service.Ports.Name
+* xRoute.Rules.Name
```

Comment:
I find this example confusing since xRoute Rules do not have a name field yet. I'm also curious why Gateway.Listeners.Name isn't listed here.

Reply:
I've added Gateway.Listeners.Name and put a caveat on the Named rules thing. Thanks!